tests/utils.py

```diff
@@ -93,17 +101,35 @@ def __init__(self, dataset_name, test_fraction):
             N = 1000
             self.X = np.random.random(size=(N, 200))
             self.y = np.random.randint(3, size=(N,))
+        elif dataset_name == "classification_rnd_w_missing_values":
+            self.name = "train_model_classification_rnd_w_missing_values"
+            N = 1000
+            self.X = np.random.random(size=(N, 200)) - 0.5
+            self.y = np.random.randint(3, size=(N,))
+            additional_test_data = np.array([
+                [np.NaN] * self.X.shape[1],
+            ])
         elif dataset_name == "classification_binary_rnd":
             self.name = "train_model_classification_binary_random_data"
             N = 1000
             self.X = np.random.random(size=(N, 200))
             self.y = np.random.randint(2, size=(N,))
+        elif dataset_name == "classification_binary_rnd_w_missing_values":
+            self.name = "train_model_classification_binary_rnd_w_missing"
```

Comment:
```suggestion
            self.name = "train_model_classification_binary_rnd_w_missing_values"
```

Reply:
I tried to fit names in a single line, hence the inconsistency in naming. I'll try your suggestions

Reply:
Ah, I forgot that we should follow that outmoded lint rule of 80 chars per line, sorry! Nevermind!

Reply:
I've already changed names using \ to break long lines. Would it be better to leave it like this or change back to shortened names?

Reply:
Oh, sorry, I didn't notice that! I'm OK with any solution.